build.gradle

```diff
@@ -693,6 +693,9 @@ if (modrinthProjectId.size() != 0 && System.getenv("MODRINTH_TOKEN") != null) {
             addModrinthDep(qual[0], qual[1], parts[1])
         }
     }
+    if (usesMixins.toBoolean()) {
```

Comment:
Should we check if modrinth/curse tokens are set?  I have a feeling the curse one might explode if it's not.

Reply:
They are, see lines 667 and 700 (now 703).

Reply:
I'm specifically worried that `addCurseForgeRelation("requiredDependency", "gtnhmixins")` will explode without a curse token set.

Reply:
```groovy
    if (usesMixins.toBoolean() && hasCurseToken()) {
        addCurseForgeRelation("requiredDependency", "gtnhmixins")
    }
```

Reply:
oh, wait, I see now; this is inside of the block that already checks that... was hard to see on github got it